src/KeyedFrames.jl

```diff
@@ -161,6 +161,24 @@ function delete!(kf::KeyedFrame, ind::Vector{<:Symbol})
     return kf
 end
 
+##### RENAME #####
+
+function rename!(kf::KeyedFrame, nms)
+    rename!(kf.frame, nms)
+
+    for (from, to) in nms
+        kf.key[kf.key .== from] = to
```

Comment:
I feel like there is a potential improvement here where we don't have to compare all of the key names on each iteration.

Reply:
Agreed. The loop comes directly from the DataFrames.jl implementation of `rename!(::Index, ...)`, and with time as tight as it is I'm going to go with @rofinn's suggestion and leave it there. Thanks!